docs/news.rst

```diff
@@ -2,6 +2,81 @@
 Release Notes
 =============
 
+2.3 (2021-02-XX)
```

Comment:
Verifying that when this goes live the "XX" will be updated with the correct day.

Reply:
Yes, I'll update with the exact release date.